index.html

```diff
@@ -5,34 +5,82 @@
 </head>
 <body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="#about">About</a></li>
+                <li><a href="#experience">Experience</a></li>
+                <li><a href="#skills">Skills</a></li>
+                <li><a href="#projects">Projects</a></li>
+                <li><a href="#connect">Connect</a></li>
+            </ul>
+        </nav>
+    </header>
+    
     <h1 id="name">Kenneth Ward</h1>
     
     <section id="about">
         <h2>About</h2>
         <p>
-            I am Kenneth Ward Jr., I'm a Bay Area Native. I currently work at GLIDE, a non-profit in San Francisco, California. I'm also a videographer; I run my own company, Junior Ward Productions, and I'm working to be a software engineer.
+            I'm Kenneth Ward, and I'm passionate about my work at the GLIDE Foundation, a non-profit in San Francisco. Through social media and visual content, I promote the impactful work GLIDE does. In addition to visual storytelling, I manage the foundation’s website—designing, updating highlights, accomplishments, and services. My career in coding is driven by a desire to enhance our web services and advance my skills. Outside of my professional role, I am a photographer and videographer who has worked independently with non-profits like Code Tenderloin, East Oakland Youth Development Centre, and Dental Robin Hood. I’ve also shot music videos and interviews.
         </p>
     </section>
     
     <section id="experience">
         <h2>Experience</h2>
+        <div class="job">
+            <div class="job-title">Social Media Manager and Digital Content Specialist at GLIDE</div>
+            <div class="job-dates">2018 - Present</div>
+        </div>
+        <div class="job-description">
+            Managed social media channels and created digital content to enhance GLIDE's online presence.
+        </div>
+
+        <div class="job">
+            <div class="job-title">Videographer at Junior Ward Productions</div>
+            <div class="job-dates">2016 - Present</div>
+        </div>
+        <div class="job-description">
+            Filmed and edited promotional videos, documentaries, and event highlights for various clients.
+        </div>
+    </section>
+    
+    <section id="skills">
+        <h2>Skills</h2>
         <ul>
-            <li>Social Media Manager and Digital Content Specialist at GLIDE</li>
-            <li>Videographer at Junior Ward Productions</li>
-            <li>Volunteer Coordinator for various community service projects</li>
-            <li>JavaScript and HTML Developer</li>
+            <li>Video Editing</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+            <li>Social Media</li>
+            <li>Analytics</li>
+            <li>Content Creation</li>
+            <li>Content Strategy</li>
+            <li>API</li>
+            <li>Photoshop</li>
+            <li>InDesign</li>
```

Comment:
These are supposed to be added by the javascript.  The skills section should be empty here.